src/main/java/com/lastcivilization/equipmentwriteservice/infrastructure/database/EntityMapper.java

```diff
@@ -0,0 +1,14 @@
+package com.lastcivilization.equipmentwriteservice.infrastructure.database;
+
+import com.lastcivilization.equipmentwriteservice.domain.view.EquipmentModel;
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+
+@Mapper
```

Comment:
proponuje uzyc '@Mapper(componentType = "spring")'

Reply:
Mam tylko componentModel o to Ci chodzilo?

Reply:
Nwm jak dziala to type, ale jesli ci chodzilo o to zeby z mapera zrobic komponent springowy to componentModel zadzialo, testy przeszlo